### PR TITLE
[res] Introduce circle schema 0.9

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce circle schema 0.9 file.

### CODE REVIEW

**Comment by seanshpark** [comment]
- copy of #13930

File: res/CircleSchema/0.9/circle_schema.fbs [hunk 1]
```diff
@@ -0,0 +1,1711 @@
+// Copyright (c) 2019~2023 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jinevening) [inline res/CircleSchema/0.9/circle_schema.fbs:1]:
```suggestion
// Copyright (c) 2019~2024 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by ys44kim** [comment]
@seanshpark 
the version of schema seems to have changed from 0.8 -> 0.9.
In what case will the version of schema be changed? (ex. if op is added?)

**Comment by hseok-oh** [comment]
>In what case will the version of schema be changed? (ex. if op is added?)

When schema is updated after release.

**Review by ys44kim — APPROVED** [review]
LGTM

**Pull request merged**